### DISCUSSION TITLE
Any suggestion on finding only pictures I've taken?

### DISCUSSION THREAD

**Discussion by cbgrey** [RhetTbull/osxphotos#510] [Q&A]
First...this project is AMAZING. Thanks to everyone who's worked on it. 

I'm looking for a way to export all images (and videos) from Apple Photos that I took vs a bunch of other stuff that has ended up in the library from image downloads. 

I've tried looking for "camera_make" == Apple which gets close but misses anything I used a different camera app for and may include downloaded images that happened to be taken with Apple cameras. 

Any suggestions?

**Comment by RhetTbull** [comment]
`--not-shared` will export only photos that aren't in shared iCloud albums. That is, only pictures in your main library (the ones you've added). If you imported photos from someone else (Eg dragged them into Photos or imported from SDCard) those would also be exported too.

I think `--not-shared` is what your looking for.

PS -- thanks for the feedback--glad you found osxphotos useful!

**Reply by cbgrey** [reply]
Thanks Rhett. Not sure that quite gets me there but I'll try it out.

**Reply by RhetTbull** [reply]
> that I took vs a bunch of other stuff that has ended up in the library from image downloads.

I reread your question and I think I understand better.  I think you'll need to use both --not-shared plus some other criteria to search by camera model.  Photos doesn't have a way (and thus doesn't track) of knowing who took a photo (except in cases of shared iCloud albums but this is really who shared the photo not who took it). 

You could use multiple criteria in form of `--query-eval "photo.exif_info.camera_make == 'Apple'"` etc
